composer.json

```diff
@@ -102,6 +102,9 @@
             },
             "drupal/address": {
                 "https://www.drupal.org/project/address/issues/3144823": "https://www.drupal.org/files/issues/2020-06-02/address_content_language-3144823-3.patch"
+            },
+            "symfony/dom-crawler": {
+                "Fix from version 4.4 https://github.com/symfony/symfony/pull/32440/files": "./patches/symfony/dom-crawler/trim_new_line_characters.patch"
```

Comment:
This is a code from the symfony/dom-crawler v.4.4. It's quite hard update package to that version because dependency "symfony/dom-crawler": "~3.4" exists in lots of projects including drupal/core-dev https://github.com/drupal/core-dev/blob/8.9.x/composer.json. Drupal 9 has dependency "symfony/dom-crawler": "^4.4 !=4.4.5"  https://github.com/drupal/core-dev/blob/9.0.x/composer.json. So we need this patch until migration to Drupal 9.
Why do we need this patch for? This patch removes new line characters that exist in lots of templates. Without it I used strings for asserts `"09199 Ages Burgos, Spain\n\n  Munich, Germany"`. With patch I can define trivial `'09199 Ages Burgos, Spain Munich, Germany'`. See https://github.com/openeuropa/oe_theme/pull/629/commits/74bcc10208e4de5fa36fdd3654f651eb8f9e7679.

symfony/dom-crawler v.4.4. already sanitise it, so we will need to rewrite test asserts in any case.

Reply:
Good catch guys, only I'm a bit uneasy to ship with a patch, as this might break builds down the line (patches from components are applied). Let's do like this:

- We keep this aside from now, and we ship with `\n` for the test
- We open an issue where we track these findings, we can create a Drupal 9 epic already (if not already there), so we fix this when we pass to Drupal 9

Reply:
@ademarco  Without this patch we also sometimes need to use different expected values in different PHP versions - https://github.com/openeuropa/oe_theme/pull/634/files#diff-45e52785ef57fd3b53dcc3e8fbb0c31c7ec218e12361a503ef37acca4b9e7bcdR240.
Explanation is here https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-437?focusedCommentId=4402494&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4402494

Reply:
I agree with @yenyasinn, the ewcms pr shows that there is no conflict with the other components, this would also bring the code base a bit closer to D9, so for me this is ok. Up to you @ademarco

Reply:
@drishu I have already rewritten code without patch.